draft-ietf-sframe-enc.md

```diff
@@ -836,81 +836,127 @@ The authors wish to specially thank Dr. Alex Gouaillard as one of the early
+```
+OverheadKbps = (1 + |CTR| + |KID| + |TAG|) * 8 * UnitsPerSecond / 1024
+```
 
-* 20ms (50 packets/s)
-* 40ms (25 packets/s)
-* 100ms (10 packets/s)
+Here the constant value `1` reflect the fixed SFrame header; `|CTR|` and
+`|KID|` reflect the length of those fields; `|TAG|` reflects the cipher
+overhead; and `UnitsPerSecond` reflects the number of SFrame-encrypted units
+sent per second (e.g., packets or frames per second).
 
-Up to 3 bytes frame counter (3.8 days of data for 20ms frame duration) and 4
-bytes fixed MAC length.
+In the remainder of this secton, we compute overhead estimates for a collection
+of common scenarios.
 
-| Counter len| Packets   | Overhead  | Overhead | Overhead  |
-|            |           | bps@20ms  | bps@40ms | bps@100ms |
-|:----------:|:---------:|:---------:|:--------:|:---------:|
-|          1 | 0-255     |      2400 |     1200 |       480 |
-|          2 | 255 - 65K |      2800 |     1400 |       560 |
-|          3 | 65K - 16M |      3200 |     1600 |       640 |
+## Assumptions
 
-## Video
+In the below calculations, we make conservative assumptions about SFrame
+overhead, so that the overhead amounts we compute here are likely to be an upper
+bound on those seen in practice.
 
-The per-stream overhead bits per second as calculated for the following video
-encodings:
+| Field           | Bytes | Explanataion                                      |
+|:----------------|------:|:--------------------------------------------------|
+| Fixed header    | 1     | Fixed                                             |
+| Key ID (KID)    | 2     | >255 senders; or MLS epoch (E=4) and >16 senders  |
+| Counter (CTR)   | 3     | More than 24 hours of media in common cases       |
+| Cipher overhead | 16    | Full GCM tag (longest defined here)               |
 
-* 30fps @ 1000Kbps (4 packets per frame)
-* 30fps @ 512Kbps (2 packets per frame)
-* 15fps @ 200Kbps (2 packets per frame)
-* 7.5fps @ 30Kbps (1 packet per frame)
+In total, then, we assume that each SFrame encryption will add 22 bytes of
+overhead.
 
-Overhead `bps = (Counter length + 1 + 4 ) * 8 * fps`
+We consider two "units" of encryption, frames and packets.  In each scenario, we
+compute the SFrame overhead in absolute terms (Kbps) and as a percentage of the
+base bandwidth.
 
-| Counter len| Frames    | Overhead   | Overhead   | Overhead   |
-|            |           | bps@30fps  | bps@15fps  | bps@7.5fps |
-|:----------:|:---------:|:----------:|:----------:|:----------:|
-|          1 | 0-255     |       1440 |       1440 |        720 |
-|          2 | 256 - 65K |       1680 |       1680 |        840 |
-|          3 | 56K - 16M |       1920 |       1920 |        960 |
-|          4 | 16M - 4B  |       2160 |       2160 |       1080 |
+## Audio
 
-## SFrame vs PERC-lite
+In audio streams, there is typically a one-to-one relationship between frames
+and packets, so the overhead is the same whether one uses SFrame at a per-packet
+or whole-frame level.
```

Comment:
We refer to this as "per-frame" later, maybe we should here too, to keep consistency with other usages, as well as the naming convention per-*.
```suggestion
or per-frame level.
```